doc/developer-guides/hld/hld-devicemodel.rst

```diff
@@ -58,7 +58,7 @@ options:
                [--enable_trusty] [--intr_monitor param_setting]
                [--acpidev_pt HID] [--mmiodev_pt MMIO_regions]
                [--vtpm2 sock_path] [--virtio_poll interval] [--mac_seed seed_string]
-               [--cpu_affinity pCPUs] [--lapic_pt] [--rtvm] [--windows]
+               [--cpu_affinity lapic_id] [--lapic_pt] [--rtvm] [--windows]
```

Comment:
More than one lapic_id can be specified here, so should we make this:
```suggestion
               [--cpu_affinity lapic_ids] [--lapic_pt] [--rtvm] [--windows]
```

Reply:
Updated.